### ISSUE TITLE
Evalulate Multiple Config Files Separately Instead of Combining them

### ISSUE THREAD

**Issue by aliasghar98** [promptfoo/promptfoo#1098]
I'm looking for a way to run `promptfoo eval` on multiple config files but without combining them together.

I noticed that when I ran the following (referring to [this issue](https://github.com/promptfoo/promptfoo/issues/329)):

`promptfoo eval -c myconfig1.yaml myconfig2.yaml`

This ended up combining the two config files which was not what I wanted.

I tried the following too:

`promptfoo eval -c configs/*`

But this ended up combining all the config files within the `configs` directory. I'm looking for a way to run `promptfoo eval` separately on all the config files within a directory but not combine them all together into a single config file. Is this possible?

**Comment by typpo** [comment]
What is the desired behavior in this situation?  Are you expecting multiple evals that you can review in the end, e.g. a single eval with different sections?

**Comment by aliasghar98** [comment]
Yes, each config file in my scenario is running a various number of tests (these tests are different in every config file) on the prompt (this prompt is also different in every file) specified within that config file, and the provider specified also varies for each config file. The existing behaviour after running: `promptfoo eval -c configs/*` results in all of these being combined. 

Creating a separate directory for each config file and then running `promptfoo eval` on each config file separately is a time-consuming task. Also, I cannot view all the results in a consolidated place that way, I would have to check each one of them separately. What I'm looking for is to have different sections corresponding to each eval (which has been run separately/independently for each file) within a single report. I hope this clears out my intention. Please let me know if you require further details.

**Comment by cheungjulia** [comment]
Was this solved? Facing a similar issue right now!

**Comment by mldangelo** [comment]
@cheungjulia this is not currently supported in promtpfoo, but you can use this one-liner to evaluate each config file separately:

```sh
find . -maxdepth 1 -name "*.yaml" | xargs -I {} sh -c 'npx promptfoo eval -c "{}"'
```

To make it reusable, add this as an alias to your .bashrc or .zshrc:

```sh
alias pfm='find . -maxdepth 1 -name "*.yaml" | xargs -I {} sh -c '\''npx promptfoo eval -c "{}"'\'''
```
An LLM can be really helpful in modifying the command to suite your exact use case (subdirectories, other args, etc).

**Comment by cheungjulia** [comment]
Thank you! Works perfectly :)

**Comment by mldangelo** [comment]
Linking to #2453 which describes a similar use case with a detailed repro. I posted a workaround there using a single placeholder provider with per-test `provider:` overrides.

**Comment by bilal-aamer** [comment]
Hi, Is this ticket being worked upon? I have the same issue
